app/helpers/find/current_point_in_recruitment_cycle_helper.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Find
+  module CurrentPointInRecruitmentCycleHelper
```

Comment:
I think these methods could just go in here? https://github.com/DFE-Digital/publish-teacher-training/blob/442ce94593a4f5529683b9829e0bd2d4abcde605/app/helpers/recruitment_cycle_helper.rb#L3

Reply:
Yup

https://github.com/DFE-Digital/publish-teacher-training/pull/3634/commits/6970d843a15022b4bc7c8a6df08b0a47e67c337d